core/utils/poly-look/src/visualisations/charts/bar-charts/verticalBarChart/verticalBarChart.js

```diff
@@ -4,6 +4,8 @@ import { Chart } from "../../chart";
 
 import "./verticalBarChart.css";
 
+export const type = "vertical-bar-chart";
```

Comment:
Also, if you use two classes in the same client, you'll need to rename `type` when importing. Anyway, if you decide to use a const, it might better be a part of the class itself. We use a function, "title" in importers

https://github.com/polypoly-eu/polyPod/blob/f9e45e95af9b566b62cd5ac00d3231054eaae927/features/facebookImport/src/model/analyses/report/inactive-cards-summary.js#L30-L32

I don't particularly like it, but depending what you're looking for, it's linked to the class/object (not the file) it might be an alternative.

Reply:
It is part of the class, just through the top-level class Chart, which also constructs the div

Reply:
I mean accessible from a class instance, not by importing the file.

Reply:
It is accessible from the class instance through Chart just by instance.type or what do you mean?

Reply:
I see, you're using the file-scoped constant to initialize the super attribute. Fair enough.

Reply:
The changes also solved this i think